blackbox/docs/administration/system-information.txt

```diff
@@ -469,6 +469,16 @@ The table schema is as follows:
 | ``os['cgroup']['cpu']['time_throttled_nanos']`` | The total time (in nanoseconds) for which tasks in   | ``LONG``    |
 |                                                 | the cgroup have been throttled.                      |             |
 +-------------------------------------------------+------------------------------------------------------+-------------+
+| ``os['cgroup']['mem']``                         | Information about memory resources used by taks in   | ``OBJECT``  |
+|                                                 | a cgroup.                                            |             |
++-------------------------------------------------+------------------------------------------------------+-------------+
+| ``os['cgroup']['mem']['control_group']``        | The path to the memory cgroup                        | ``STRING``  |
++-------------------------------------------------+------------------------------------------------------+-------------+
+| ``os['cgroup']['mem']['usage_bytes']``          | The total current mememory usage by processes in     | ``STRING``  |
+|                                                 | the cgroup.                                          |             |
++-------------------------------------------------+------------------------------------------------------+-------------+
+| ``os['cgroup']['mem']['limit_bytes']``          | Lists the max. amount of user memory in the cgroup.  | ``STRING``  |
```

Comment:
I would remove . from max. Also user or used? How about phrasing the same as above: ...memory used by processes in the cgroup.